lib/context.js

```diff
@@ -1,13 +1,23 @@
 import { readFile, writeFile } from 'mz/fs'
 import { resolve } from 'path'
 import { homedir } from 'os'
+import findUp from 'find-up'
 
 import { proxyStringToObject } from './utils/proxy'
 
 let context
 
-export function getConfigPath () {
-  return resolve(homedir(), '.contentfulrc.json')
+export async function getConfigPath () {
+  const contentfulrc = '.contentfulrc.json'
+  try {
+    const configPath = await findUp(contentfulrc)
+    if (!configPath) {
+      throw Error('no config path found')
```

Comment:
Is there any reason we changed this to throw if we don't find the path, we can maybe send back `resolve(homedir(), '.contentfulrc.json')` if we don't find anything

Reply:
One other thing maybe we can cache the path once we found it, at lease while the process is running.

Reply:
hm good q. I guess I was trying to cover the case where (1) findUp throws an error and (2) findup finds nothing (aka resolves with `null`) both in the same catch statement.

I'll follow up about caching in person

Reply:
we chatted in person all resolved 👍